userdata/rundsc.ps1

```diff
@@ -367,6 +377,23 @@ function New-LocalCache {
   }
 }
 
+# SourceRepo is in place to toggle between production and testing environments
+$evn:SourceRepo = mozilla-releng
```

Comment:
was this meant as $SourceRepo or maybe $env:SourceRepo ? I don't think it's doing anything as is.